etl/steps/data/garden/health/2022-12-28/karlinsky.meta.yml

```diff
@@ -0,0 +1,50 @@
+dataset:
+  sources:
+  #   -
+  #     name: Maddison Project Database 2020 (Bolt and van Zanden, 2020)
+  #     published_by: Bolt, Jutta and Jan Luiten van Zanden (2020), “Maddison style estimates of the evolution of the world economy. A new 2020 update“.
+  #     publisher_source: |
+  #       The Maddison Project Database is based on the work of many researchers that have produced estimates of...
+
+tables:
+  deaths_karlinsky:
+    title: International Completeness of Death Registration 2015-2019 – Karlinsky (2021)
+    description: >
+      Contains all metrics used in the International Completeness of Death Registration 2015-2019 paper.
+    variables:
+      # Life expectancy
+      reg_deaths:
+        title: Registered deaths
+        unit: "deaths"
+        description: >
+          The number of deaths registered in the vital registration system for the country-year. NA denotes no registered deaths data was obtained.
+      expected_wpp:
+        title: Expected number of deaths (WPP 2019)
```

Comment:
For the sake of future maintenance, I'd suggest removing years in the titles and descriptions of variables. Otherwise, you will need to manually update them. I think they are not necessary, but if they are, then keep them..

Reply:
This is relevant, as the estimations by WPP, GBD and GHE can differ substantially between revisions. The user should know which version is being used. Even more, in this case, where COVID had not yet happened.

Also, we will need to create the metadata file again when we revise the version. With newer versions new columns might even appear. I think this is precisely one of the reasons why `snapshot`->`meadow` makes more sense vs. `etag`